src/demos/calcite-radio-button-group.html

```diff
@@ -48,6 +52,58 @@
     <calcite-button href="/">
       < Home</calcite-button>
         <h1>Calcite Radio Button Group</h1>
+
+        <section>
+          <!-- src/components/calcite-radio-button-group/usage/.md -->
+          <div>
+            <h3>Basic</h3>
+            <calcite-radio-button-group name="group-basic">
+              <calcite-radio-button value="maps" checked>Maps</calcite-radio-button>
+              <calcite-radio-button value="layers">Layers</calcite-radio-button>
+              <calcite-radio-button value="data">Data</calcite-radio-button>
+              <calcite-radio-button value="geography">Geography</calcite-radio-button>
```

Comment:
☝️ same here, these should be updated to have wrapping `calcite-label`s.

Reply:
(also done)